packages/credential-showcase-traction-adapter/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "credential-showcase-traction-adapter",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc",
+    "build:clean": "tsc --build --clean && tsc --build",
+    "prettier": "prettier --write \"{packages,__tests__,!dist}/**/*.{ts,js,json,md}\"",
+    "test:ci": "jest --config=jest.json",
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --verbose --config=jest.json --coverage=true --detectOpenHandles"
+  },
+  "dependencies": {
+    "express": "^4.21.2",
+    "rhea": "^3.0.3",
+    "rhea-promise": "^3.0.3",
+    "typedi": "^0.10.0"
+  },
+  "devDependencies": {
+    "testcontainers": "^10.18.0",
+    "@testcontainers/rabbitmq": "^10.18.0",
+    "@types/express": "^5.0.0",
+    "@types/node": "^22.13.1",
+    "cross-env": "^7.0.3",
```

Comment:
alot of these dev dependencies can be removed as they are already in the main package.json

Reply:
for dev you do not

Reply:
But we can't run jest on the local package anyway, we always set the monorepo root as workdir where the jest.json is and just set the test file or dir to ./packages/xyz
For local building all packages are hoisted from the monorepo root, so there is no issue there.
Having duplicate deps just increases the risk of having mixed versions IMO

Reply:
fixed

Reply:
> Having duplicate deps just increases the risk of having mixed versions IMO

exactly